sos/policies/redhat.py

```diff
@@ -45,6 +45,7 @@ class RedHatPolicy(LinuxPolicy):
     _in_container = False
     _host_sysroot = '/'
     default_scl_prefix = '/opt/rh'
+    name_scheme = 'friendly'
```

Comment:
Is there a reason to have non-default `name_scheme` for one specific policy? I understand your intention, but I dont understand what is specific on the scheme for different policies.

(maybe have this as a user-configured option? or rather directly have sosreport option --report-namescheme="-{name}-{case}-%Y-%m-%d-{rand}" that will be appended after "`sosreport`" string?)

Reply:
I am wanting to change the RH naming scheme because that's where I have a pulse with people using sosreports on a daily basis. I can't say the same for any other OSes, so I left those alone.

I think this should be policy driven because it's unlikely (in my opinion) that a user would actively change the pattern via individual runs of sos because that would be a bit clunky.